src/Provider/OrientedImageProvider.js

```diff
@@ -0,0 +1,35 @@
+import format from 'string-format';
+import Fetcher from './Fetcher';
+
+/**
+ * @module OrientedImageProvider
+ */
+/**
+ * Request textures for an oriented image. It is used by {@link OrientedImageLayer} to request a set of texture for a given panoramic.
+ * @param {Promise} command - Description of the panoramic that need textures.
+ * @return {Promise} Promise that resolve with an array of textures.
+ */
+function executeCommand(command) {
+    const layer = command.layer;
+    const pano = command.requester;
+    if (pano != layer.currentPano) {
+        // command is outdated, do nothing
+        return Promise.resolve();
+    }
+    // prepare a promise for each texture to load.
+    var promises = [];
+    // panoramic id
+    const imageId = pano.geometry[0].properties.id;
+    // request one texture for each camera of the panoramic
+    for (const camera of layer.cameras) {
+        var sensorId = camera.name;
+        var url = format(layer.source.url.images, { imageId, sensorId });
```

Comment:
In the logic of itowns, `url` should be built and provided by the `source`.

Reply:
ok